src/SonarScanner.MSBuild.Tasks/Targets/SonarQube.Integration.targets

```diff
@@ -384,7 +384,7 @@
                          OriginalAnalyzers="@(Analyzer)"
                          OriginalAdditionalFiles="@(AdditionalFiles)"
                          OriginalRulesetFilePath="$(ResolvedCodeAnalysisRuleset)"
-                         ProjectSpecificOutputDirectory="$(ProjectSpecificOutDir)">
+                         ProjectSpecificConfigDirectory="$(ProjectSpecificConfDir)">
```

Comment:
The merged ruleset is a configuration file rather than an output file, so it should be written to the _conf_ rather than the _out_ directory.